cdc_rsync/cdc_rsync.h

```diff
@@ -46,7 +46,9 @@ struct Options {
   const char* copy_dest = nullptr;
   int compress_level = 6;
   int connection_timeout_sec = 10;
-
+  const char* ssh_command = nullptr;
```

Comment:
Is there a reason to not use `absl::string_view`?

Reply:
In this incarnation, the data is passed through a DLL boundary, so one needs to use POD types. absl::string_view is probably aright, but probably also not a good idea, e.g. if two different ABSL versions would be used between DLLs and they changed something about the type. Anyway, in a later CL I removed the DLL and used std::strings, which is necessary since some vars can come from environment vars (CDC_SSH_COMMAND).